rules/const.bzl

```diff
@@ -71,6 +71,29 @@ CONST = struct(
     ),
 )
 
+def get_lc_items(*want_lc_values):
+    """Return a list of key-value pairs from CONST.LCV with lowercased keys.
+
+    If `want_lc_values` is empty, returns an unfiltered dict. Otherwise, only
+    key-value pairs where the value is in `want_lc_values` are returned.
+    """
+    lcv_dict = structs.to_dict(CONST.LCV)
+
+    for want_val in want_lc_values:
+        if want_val not in lcv_dict.values():
+            fail("Unrecognized LCV: {}".format(want_val))
+
+    out = []
+    for key, val in lcv_dict.items():
+        key = key.lower()
+        if len(want_lc_values) == 0 or val in want_lc_values:
+            out.append((key, val))
+
+    if len(want_lc_values) != 0 and len(out) == 0:
+        fail("get_lc_dict would produce an empty list")
+
+    return out
```

Comment:
A few questions/comments:
* I haven't see the rest of the PR, so this might be a silly question but why does this function return a list instead of a dict? I guess once we grab the states we want, we don't need to perform lookups, right?
* We can also revisit the capitalization of the fields of `CONST`. I think we capitalized them for consistency but I'm not opposed to using lower-case letters if upper-case letters make our lives harder.
* This seems to be useful as a generic `dict` function, e.g. `get_dict_items(my_dict, *want_items)`. We don't have to do it in this PR, just something to think about.
* Some simplifying suggestions (I hope I didn't miss anything):
```suggestion
def get_lc_items(*want_lc_values):
    """Return a list of key-value pairs from CONST.LCV with lowercased keys.

    If `want_lc_values` is empty, returns an unfiltered dict. Otherwise, only
    key-value pairs where the value is in `want_lc_values` are returned.
    """
    lcv_dict = structs.to_dict(CONST.LCV)
    out =  [(k.lower(), v) for k, v in lcv_dict.items() if v in want_lc_values or !want_lc_values]
    if want_lc_values and len(out) != len(want_lc_values):
        fail!("Check your keys!")
    return out
```

Reply:
> A few questions/comments:
> 
> * I haven't see the rest of the PR, so this might be a silly question but why does this function return a list instead of a dict? I guess once we grab the states we want, we don't need to perform lookups, right?

It's just that a dict would be less useful because then we'd need to use `.items()` at every call site. And as you pointed out, we don't need to look anything up by keys, so a list is more natural.

> * We can also revisit the capitalization of the fields of `CONST`. I think we capitalized them for consistency but I'm not opposed to using lower-case letters if upper-case letters make our lives harder.

I like the capitalization for indicating that the values are constant. But it does get in the way a bit when squishing it into a Bazel target name. If we normalize on this function, I think we entirely avoid the pain point :)

> * This seems to be useful as a generic `dict` function, e.g. `get_dict_items(my_dict, *want_items)`. We don't have to do it in this PR, just something to think about.

Agreed, although the odd thing about this function is that it filters on the values rather than the keys. Let's wait to generalize the name until we have multiple callers that want the abstraction :) 

> * Some simplifying suggestions (I hope I didn't miss anything):

I think this is equivalent, though the value check is a little less obvious now. Much shorter, thanks!